setuptools/tests/test_easy_install.py

```diff
@@ -71,19 +71,34 @@ def test_install_site_py(self, tmpdir):
 
     def test_get_script_args(self):
         header = ei.CommandSpec.best().from_environment().as_header()
-        expected = header + DALS(r"""
-            # EASY-INSTALL-ENTRY-SCRIPT: 'spec','console_scripts','name'
-            __requires__ = 'spec'
-            import re
-            import sys
-            from pkg_resources import load_entry_point
-
-            if __name__ == '__main__':
-                sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
-                sys.exit(
-                    load_entry_point('spec', 'console_scripts', 'name')()
-                )
-            """)  # noqa: E501
+        if sys.version_info >= (3, 8):
```

Comment:
I'll plan to consolidate this fork later... and just make some assertions about the result that are common to both behaviors.